features/steps/consistency.py

```diff
@@ -75,9 +75,9 @@ def step_impl(context, image, output_format):
     output_split = output_switch.split(' ')
     # there's probably a more pythonic way to do this, with array flattening etc.
     # just being expedient for the time being
-    args = [found_executable, '-tiff', output_split[0], image]
+    args = [found_executable, '-tiff', '-color', 'none', '-no-denoise', output_split[0], image]
     if len(output_split) == 2:
-        args = [found_executable, '-tiff', output_split[0], output_split[1], image]
+        args = [found_executable, '-tiff', '-color', 'none', '-no-denoise', output_split[0], output_split[1], image]
```

Comment:
I will take a look. My python experience is more or less as extensive as
yours though :)

split() seems a bit unnecessary. You could just define output_switch a
list instead.

On sáb, 2015-07-25 at 16:32 -0700, Roland Karlsson wrote:
> In features/steps/consistency.py:
> 
> >      if len(output_split) == 2:
> > -        args = [found_executable, '-tiff', output_split[0], output_split[1], image]
> > +        args = [found_executable, '-tiff', '-color', 'none', '-no-denoise', output_split[0], output_split[1], image]
> I have done something similar. You have to take into account that you
> shall have the switch -no-crop if you do NOT want to crop. I am just
> running a test to see if I get no errors. I did get an error when I
> wanted to crop, but I think I can fix it. please take a look if you
> like it when I have pushed it it is the first python I ever write :)
> The reason for the error was that ''.split(' ') do not return the
> empty list: [] but a list with an empty string: ['']
> 
> —
> Reply to this email directly or view it on GitHub.
> 
>